src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -185,7 +185,12 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			return;
 		}
 		const dialog = this._instantiationService.createInstance(TableDesignerPublishDialog);
-		const result = await dialog.open(report);
+		let result: TableDesignerPublishDialogResult;
+		if (typeof report === 'string') {
+			result = await dialog.open(report);
+		} else {
+			result = await dialog.open(report[0], report[1]);
```

Comment:
Using indexes here is a bit odd - why aren't you using the property names (report and format)?

Reply:
does this actually work? you have to do report.content or report.format, right?